server/routes/recat.js

```diff
@@ -38,7 +38,7 @@ module.exports = function Index({
     '/securityInput/:bookingId',
     asyncMiddleware(async (req, res, transactionalDbClient) => {
       const { bookingId } = req.params
-      const result = await buildFormData(res, req, 'ratings', 'securityInput', bookingId, transactionalDbClient)
```

Comment:
Should stay 'ratings' IMO. As there is already data under that field name.

Reply:
here we are building new for data for recats and we generally use ratings for initial cat.